tdvt/tdvt/exprtests/standard/expected.setup.cast.str.datetime.1.txt

```diff
@@ -0,0 +1,120 @@
+<results>
+  <test name='DATETIME(&apos;1900-01-01 01:00:00&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3780030237)(0)]</column>
```

Comment:
We want to use hyper to generate the base reference, unless there's a reason to use a different ds.

Reply:
We had to create this expected based on sqlserver as for some of the datasources seconds value is not set after cast